app/index.py

```diff
@@ -38,13 +37,13 @@ def get(self, conn, bbox: Tuple[float, float, float, float]):
         """Get the featureIDs in the `bbox`. BBOX comparison is string comparison of
         the coordinate values that are formatted to three decimal places.
         """
-        # We exepect that at this point we have a valid 'bbox', as in a tuple of
-        # four floats
+        # We expect that at this point we have a valid 'bbox', 
+        # as in a list of four floats.
```

Comment:
The function signature requires a tuple, not a list. Is there a specific reason why we would need a list here instead of a tuple?